spec/mailers/activity_mailer_spec.rb

```diff
@@ -1,9 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe ActivityMailer, type: :mailer do
-  let(:template_path) { File.join(__dir__, '../fixtures/activity_mailer/test_email') }
+  let(:template_path) { '../../spec/fixtures/activity_mailer/test_email' }
```

Comment:
hmm... so sorry I still do not quite understand what you mean. And I can not just pass a template name into this mailer. Could you please give me some suggestions? Thanks